src/App.js

```diff
@@ -1,30 +1,27 @@
-import logo from './logo.svg';
-import './App.css';
+import Button from './components/Button';
+
 
 function App() {
   return (
     <div>
       <div>
-
-      
-        <div className='logo'>
+        <div className="logo">
           <img
             src="https://consider-it.de/wp-content/uploads/2024/06/CIT-horizontal-V1-20240221.svg"
             alt="Logo"
           />
         </div>
-        <div className='demo-text'>
+
+        <div className="demo-text">
           <p>Demo-text</p>
         </div>
+        
       </div>
-      <div className='field-text'>
-          <input type='text' className='input-text'/>
-      </div>
-
-      <div className='div-go-button'>
-        <button className='go-button'>GO</button>
+      <div className="field-text">
+        <input type="text" className="input-text" />
       </div>
 
+      <Button label={"ok"}/>
```

Comment:
Du hast in dem Button Component ebenfalls ein `onClick` prop definiert. Diesen benutzt du aber noch nicht. Du könntest beim click sowas sagen wie `console.log("button clicked...")`.